src/components/SingleComment.jsx

```diff
@@ -0,0 +1,10 @@
+function SingleComment({ comment }) {
+  return (
+    <div>
+      <strong>{comment.author}</strong>
```

Comment:
I would add some styling to this - its not clear what the votes are for right now on the page